### ISSUE TITLE
Mechanical arm not taking items from belt

### ISSUE THREAD

**Issue by T-Dark0** [Fabricators-of-Create/Create#327]
### Describe the Bug

Mechanical arms don't appear to take items from belts. They can still deposit to belts, but not extract from them

### Reproduction Steps

1. Place a mechanical arm. Link it to a belt as source and a depot as output (Also tested with a brass funnel, both filtered and unfiltered, both pointing to a vanilla barrel and a mechanical crafter)
2. Power the arm
3. Witness it not do anything

### Expected Result

I expected the arm to function, taking items from the belt and putting them onto the depot/into the funnel.

### Screenshots and Videos

![2022-05-28_21 05 56](https://user-images.githubusercontent.com/60000771/170839530-fc0c88ed-4047-4f34-9f06-39ed72a16a0e.png)

### Operating System

Windows 10

### Mod Version

0.4.1

### Minecraft Version

1.18.2

### Other Mods

A modified version of All of Fabric 5 version 1.2.0, with the addition of Paginated Advancements and the removal of:
- Advancements Enlarger
- Adventurez
- AOF Emotes
- Better Bad Omen
- Bosses Of Mass Destruction
- Capybara
- Caracal
- Companion
- Creeper Overhaul
- Dusk
- Dwarf Coal
- Ecologics
- Goblin Traders
- Lovely Snails
- Magitek Meks
- Macaw's Bridges
- Macaw's Paintings
- Mythic Mounts
- Whisperwoods

**Comment by Sheep7798** [comment]
I have the same problem but I thought it was me being stupid lol

**Comment by TropheusJ** [comment]
if you relog / restart the server does it magically start working? likely #244

**Comment by Sheep7798** [comment]
Unfortunately it didn't. I tried everything but it didn't work on my side. To be sure that I had done no errors, I took my world and I loaded it on the forge version of create and the mechanical arm took the items from the belts (it didn't on fabric version).  Now, relogging indeed fixes a bug (https://github.com/Creators-of-Create/Create/issues/2968), but the mechanical arm still doesn't take the items from belts.
